medusa/init/filesystem.py

```diff
@@ -9,6 +9,7 @@
 import tarfile
 import tempfile # noqa # pylint: disable=unused-import
 
+import tempfile # noqa # pylint: disable=unused-import
```

Comment:
Duplicate import?

Reply:
I think this is not mine, but  looks like a typo in "master" branch...

Reply:
Done, removed.